### ISSUE TITLE
Spurious wideband signals in Waterfall and Audio when using higher sampling rates and/or RX2

### ISSUE THREAD

**Issue by dividebysandwich** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#597]
SunSDR2DX
Serial: EED06462200939
Firmware: 72.8
EESDR ver 1.0.4 beta
Build 24042023
Windows 10

I have a problem with spurious signals using both receivers coupled with problems operating with different sample rate settings. The default of 39kHz works fine with a single RX, but as soon as I start using RX2 there's wild stutters and interference on both bandscopes and the audio. 

![image](https://github.com/ExpertSDR3/ExpertSDR3-BUG-TRACKER/assets/23048489/1c900419-9389-4409-9529-df4ed11b30c4)

This somewhat goes away if I select the highest sample rate of 312kHz, but not completely. What happens then is that I get very short spurious wideband bursts (almost look like lightning) that kill the AGC every few seconds. However these signals are not physically present.

![image](https://github.com/ExpertSDR3/ExpertSDR3-BUG-TRACKER/assets/23048489/2a7b7a20-4f02-403b-ac38-aacd3889a312)

So to summarize: At 39kHz I can use RX1, but not both RX1 and RX2. At 312kHz I can use both RX1 and RX2 but there's still problems with the signals. The intermediate sampling rates produce all kinds of different weird interference.
Notably this does not only affect the audio, but the whole waterfall too. I have tried the improve_udp registry change, my audio device has exclusive mode turned off, I have a Ryzen 5950X with 32GB RAM, and I do have my LAN interface with the lowest metric.

None of these problems occur with ExpertSDR2. There I can use RX1 and RX2, at any sample rate with no issue.

**Comment by BuddyButterfly** [comment]
Does a firmware upgrade help?
I am using firmware 86.8 with ESDR3 as can be seen in the list of #592, whereas you are using 72.8?

**Comment by dividebysandwich** [comment]
Was that a recent release? I am certain I saw no available firmware update like 2 days ago, and now it popped up when I started EESDR3. I did the update and the problems are gone - I can now use both receivers in any sample rate, no issues!
Thanks for the hint!

**Comment by BuddyButterfly** [comment]
Welcome! If all would be fixed that quickly ;-)
